README.rst

```diff
@@ -5,38 +5,21 @@ Clear Containers Tests Repository
 
 Functional tests
 ----------------
-
-In order to run all functional test you have to install next packages
-
-- github.com/onsi/ginkgo/ginkgo
-- github.com/onsi/gomega
-
-To install above packages, just run::
-
-  $ go get github.com/onsi/ginkgo/ginkgo
-  $ go get github.com/onsi/gomega
-
-and add ``$GOPATH/bin`` to the ``$PATH``::
-
-  $ export GOBIN=$GOPATH/bin
-  $ export PATH=$PATH:$GOBIN
-
 To run all functional tests, run::
 
   $ make functional
 
-`Environment variables`_ can be used to override the paths of Clear Containers components.
-For example::
 
-  $ RUNTIME="cc-runtime" make functional
+By default the functional tests use the version of ``cc-runtime`` present in the environment variable ``PATH``
```

Comment:
I am confused because I think the environment variable is called ``RUNTIME``. At least that is the variable added under "Environment variables"
Please change to:
By default, the functional tests use the version of ``cc-runtime`` set in the environment variable ``RUNTIME``